src/components/ArtObjectPageComponents/TabbedSubMenu/index.jsx

```diff
@@ -22,80 +28,116 @@ class TabbedSubMenu extends Component {
         title: 'Details',
         slug: 'details',
       },
-    ];
+    ]
 
-    this.state = { tabs: tabList };
+    this.state = { tabs: tabList }
+  }
+
+  componentDidMount() {
+    // this should always find a value
+    const selectedIndex = this.state.tabs.map(function(tab) {
+      return tab.slug
+    }).indexOf(this.props.slug)
+
+    const options = {
+      // align the first one to the left, otherwise, center it.
+      cellAlign: selectedIndex === 0 ? 'left' : 'center',
+      initialIndex: selectedIndex,
+      accessibility: true,
+      pageDots: false,
+      prevNextButtons: false,
+      wrapAround: false,
+    }
+
+    // note: #flickityWorkaround - because we can't use react-flickity without changing webpack configs
+    this.flickityInstance = new Flickity( '.tabs-flickity', options)
+  }
+
+  componentWillUnMount() {
+    // note: #flickityWorkaround - because we can't use react-flickity without changing webpack configs
+    if (this.flickityInstance) {
+      this.flickityInstance.destroy()
+    }
+
+    this.flickityInstance = null
```

Comment:
should this be in the if statement?

Reply:
yeah it could be, but it's fine outside. Because only the .destroy method needs to be protected from an error in the if statement.